.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "partiql-conformance-tests/partiql-tests"]
```

Comment:
We'll need to document somewhere whatever steps needed to be taken to pull down the test submodules (`git submodule update`) in additon to just the repo clone.